projects/govuk_publishing_components/docker-compose.yml

```
@@ -20,6 +20,14 @@ services:
   govuk_publishing_components-lite:
     <<: *govuk_publishing_components
 
+  govuk_publishing_components-jasmine:
+    <<: *govuk_publishing_components
+    environment:
+      VIRTUAL_HOST: localhost # Jasmine doesn't like non localhost domains
+    expose:
+      - "8888"
```

```suggestion
    ports:
      - "8888:8888"
```

Setting VIRTUAL_HOST requires a dependency on nginx-proxy (that's the app that picks up the VIRTUAL_HOST env var) I imagine this worked for you without setting that in depends_on because nginx-proxy was already running in the background.

I think it'd be better to use ports rather than expose though as we'll be to access the Jasmine tests via http://localhost:8888 which is the same URL that Jasmine will suggest and won't require someone to have the extra knowledge of how to access the tests.

---

Super - thanks for this. Agree about using Jasmine's default port, I just didn't know that it could be done. I've added this to the commit 👍